src/runtime_src/xdp/profile/plugin/ocl/ocl_profiler.cpp

```diff
@@ -756,8 +759,7 @@ namespace xdp {
 
     dInt->resetTS2MM();
 
-    auto addr = xrtDevice->map(mDDRBufferForTrace);
-    munmap(addr, mDDRBufferSz);
+    xrtDevice->map(mDDRBufferForTrace);
```

Comment:
Why is map called when the returned ptr is not used?

Reply:
XRT PR build/test failure seemed to be some setup/infra issue. I wanted to confirm and trigger a rebuild. I have removed it now.